### PR TITLE
feat(indexers): add Locadora

### PR DESCRIPTION
**Pull request by costaht**
That file adds a new indexer to autobrr

### CODE REVIEW

File: internal/indexer/definitions/locadora.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      required: true
+      label: Nick
+      help: Bot nick. Eg. user_bot
+
+  parse:
+    type: single
+    lines:
+      - test:
+          - "[Filmes] [WEB-DL] [1080p] [Journey to the Center of the Earth 2008 1080p HMAX WEB-DL DD5.1 x264 pt-BR ENG-LCD] [https://locadora.cc/torrents/16896] [5.82 GiB] [100%]"
+        pattern: '\[(.+)\] \[(.+)\] \[(.+)\] \[(.+?)\] \[(https?\:\/\/.+\/).+\/(\d+)\] \[(.+?)\] \[(.+?)\]'
```
Comment (s0up4200) [inline internal/indexer/definitions/locadora.yml:43]: Will the freeleech part be announced if 0%?

File: internal/indexer/definitions/locadora.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+---
+#id: Locadora 
+name: Locadora
+identifier: Locadora
```
Comment (s0up4200) [inline internal/indexer/definitions/locadora.yml:4]: identifier should be lowercaps I think!

File: internal/indexer/definitions/locadora.yml (hunk 1 above)
Comment (martylukyy) [reply internal/indexer/definitions/locadora.yml:43]: Is there a possibility that a field will be empty? For example just a `[]`?

Comment (costaht) [reply internal/indexer/definitions/locadora.yml:43]: No, we only have video content and all vars will have something in it

File: internal/indexer/definitions/locadora.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  channels:
+    - "#announce"
+  announcers:
+    - LCD
+  settings:
+    - name: nick
+      type: text
+      required: true
+      label: Nick
+      help: Bot nick. Eg. user_bot
```
Comment (zze0s) [inline internal/indexer/definitions/locadora.yaml:36]: To still give the possibility to set a nickserv account from the start add this:

```yaml

    - name: auth.account
      type: text
      required: false
      label: NickServ Account
      help: NickServ account. Make sure to group your user and bot.

    - name: auth.password
      type: secret
      required: false
      label: NickServ Password
      help: NickServ password
```

**Review by zze0s — CHANGES_REQUESTED** [review]
Looks good, just minor comment about IRC settings.

**Review by costaht — COMMENTED** [review]
Sure,

**Review by zze0s — APPROVED** [review]
Looks good! Thanks for your contribution!

**Comment by zze0s** [comment]
Related #1

**Pull request merged**